src/controller/python/ChipDeviceController-ScriptBinding.cpp

```diff
@@ -222,6 +225,15 @@ chip::Controller::Python::StorageAdapter * pychip_Storage_GetStorageAdapter()
 
 ChipError::StorageType pychip_DeviceController_StackInit()
 {
+    ReturnErrorOnFailure(chip::Platform::MemoryInit().AsInteger());
+
+    auto err = chip::DeviceLayer::PlatformMgr().InitChipStack();
```

Comment:
Is that required even? It seems that `DeviceControllerFactory::InitSystemState` (called by `DeviceControllerFactory::Init`) calls it anyways (when `CONFIG_DEVICE_LAYER` is set, which it is in our case from what I can tell).

Reply:
If we're going to share this with the server-side bits of initialization as well, then I think it does need to be there.

Reply:
It shouldn't hurt, so fine by me.